src/lang-ex.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @license
+ * Copyright (C) 2017 Google Inc.
```

Comment:
Unless you're a Google employee, the copyright should be attributed to you and by putting that license on the file, you're saying you're ok with distributing your work under that license.